package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "jupyter",
     "displayName": "Jupyter",
-    "version": "2021.6.1-rc",
+    "version": "2021.6.99-rc",
```

Comment:
I don't think this will work. we have code that updates this with the build number.
Please check the generated VSIX